clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -941,21 +881,108 @@ class KernelObjVisitor {
   // FIXME: Can this be refactored/handled some other way?
   template <typename ParentTy, typename... Handlers>
   void VisitStreamRecord(const CXXRecordDecl *Owner, ParentTy &Parent,
```

Comment:
Yeah, there are a lot of them outside of your changes :(
```suggestion
  void visitStreamRecord(const CXXRecordDecl *Owner, ParentTy &Parent,
```